app/components/projects_page.py

```
@@ -1150,27 +1470,65 @@ def new_project_modal() -> rx.Component:
+                width="100%"
+            )
         ),
         max_width="1400px",
         p=6,
-        
+
         # Modals
         children=[
-            new_project_modal()
+            new_project_modal(),
+            new_task_modal(),
```

Ruff (0.13.1)

Undefined name `new_task_modal`

(F821)